### ISSUE TITLE
Please unpin dependencies

### ISSUE THREAD

**Issue by mirekphd** [DataCanvasIO/DeepTables#46]
Please unpin python dependencies from `pkg==ver` to the usual `pkg=>ver`.

For instance the pinning of `catboost` in [requirements.txt](https://github.com/DataCanvasIO/DeepTables/blob/master/requirements.txt#L6) to an obsolete version 0.20.2 causes `pip` to downgrade this package after installing `deeptables` and our CICD scripts concludes that `catboost` needs an upgrade, leading to unnecessary rebuild of our python containers during every version check, i.e. every hour.

```
16:35:32  catboost :
16:35:32  - version installed:  0.20.2
16:35:32  - latest  available:  0.24.3
16:35:32  - package upgradeable:  True 
```

```
16:44:20  Collecting catboost
16:44:20    Downloading catboost-0.24.3-cp38-none-manylinux1_x86_64.whl (66.2 MB)
[...]
16:44:25  Collecting deeptables
16:44:25    Downloading deeptables-0.1.12-py3-none-any.whl (2.2 MB)
16:44:25  Requirement already satisfied: lightgbm in /opt/conda/lib/python3.8/site-packages (from causalml->-r /tmp/python-packages/pypi-packages.txt (line 12)) (3.0.0)
16:44:25  Collecting catboost
16:44:25    Downloading catboost-0.20.2-cp38-none-manylinux1_x86_64.whl (63.9 MB)
1
```
As you see from the log above, the other GBDT dependency, `lightgbm`, does not cause this issue (even though it is pinned to a previous version 3.0.0 on our side), because it is not pinned in your requirements file [here](https://github.com/DataCanvasIO/DeepTables/blob/master/requirements.txt#L7).

**Comment by jackguagua** [comment]
@mirekphd  Thanks for your feedback. It has been fixed.

**Comment by mirekphd** [comment]
It worked, thank you! 

```
21:02:23  catboost :
21:02:23  - version installed:  0.24.3
21:02:23  - latest  available:  0.24.3
21:02:23  - package upgradeable:  False 
21:02:23  
[..]
21:02:23  deeptables :
21:02:23  - version installed:  0.1.13
21:02:23  - latest  available:  0.1.13
21:02:23  - package upgradeable:  False 
```
I suppose we can safely close the issue now.
